scipy/signal/tests/test_signaltools.py

```diff
@@ -80,9 +80,10 @@ def test_complex(self, xp):
         z = convolve(x, y)
         xp_assert_equal(z, xp.asarray([2j, 2 + 6j, 5 + 8j, 5 + 5j]))
 
+    @xfail_xp_backends("jax.numpy", reason="wrong output dtype")
     def test_zero_rank(self, xp):
-        a = 1289
-        b = 4567
+        a = xp.asarray(1289)
+        b = xp.asarray(4567)
```

Comment:
I'd prefer to keep testing python scalars if possible.

Reply:
I've duplicated the test, one for xp scalars and another for pure python.